### ISSUE TITLE
use pretrained Places365, The results are the same for all images

### ISSUE THREAD

**Issue by aTunass** [pranavphoenix/WaveMix#3]
Hello author, i'm tried to use your pretrained model in Places365 Dataset:
_import torch, wavemix
from wavemix.classification import WaveMix
model = WaveMix(
    num_classes = 365,
    depth = 12,
    mult = 2,
    ff_channel = 256,
    final_dim = 256,
    dropout = 0.5,
    level = 2,
    initial_conv = 'pachify',
    patch_size = 8
)
url = 'https://huggingface.co/cloudwalker/wavemix/resolve/main/Saved_Models_Weights/Places365/places365_54.94.pth'
model.load_state_dict(torch.hub.load_state_dict_from_url(url))_
and when i give many images to model, i always get result:
_top_values, top_indices = torch.topk(preds, 5)
print("top_values:", top_values)
print("top_indices :", top_indices)_
top_values: tensor([[2.0731, 1.9153, 1.7019, 1.5919, 1.5876]], device='cuda:0')
top_indices: tensor([[ 12,  67, 270, 103, 317]], device='cuda:0')
Did I do something wrong at any step?

**Comment by pranavphoenix** [comment]
I need to the see more code to tell anything
